code/src/components/movieThumb.css

```diff
@@ -0,0 +1,33 @@
+.movie-thumb {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    align-items: flex-end;
+}
+
+.movie-thumb-wrapper {
+    position: absolute;
+    z-index: 1;
+    text-decoration: none;
+    padding: 20px;
+    max-width: 350px;
+    word-wrap: break-word;
+    display: none;
+}
+.movie-thumb-image:hover + .movie-thumb-wrapper {
+    display: block;
+}
+.movie-thumb-image:hover {
+    filter: brightness(30%);
+    cursor: pointer;
+}
+
+
+.movie-thumb-image {
```

Comment:
Right now your images doesn't have proper dimensions (they are too big). One more line of code and your site will look very well - `width: 100%`